op-e2e/system/fees/fees_test.go

```diff
@@ -78,6 +78,38 @@ func TestFees(t *testing.T) {
 		cfg.DeployConfig.L2GenesisFjordTimeOffset = new(hexutil.Uint64)
 		testFees(t, cfg)
 	})
+
+	t.Run("isthmus without operator fee", func(t *testing.T) {
+		op_e2e.InitParallel(t)
+		cfg := e2esys.DefaultSystemConfig(t)
+		cfg.DeployConfig.L1GenesisBlockBaseFeePerGas = (*hexutil.Big)(big.NewInt(7))
+
+		cfg.DeployConfig.L2GenesisRegolithTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisCanyonTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisDeltaTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisEcotoneTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisFjordTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisHoloceneTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisIsthmusTimeOffset = new(hexutil.Uint64)
+		testFees(t, cfg)
+	})
+
+	t.Run("isthmus with operator fee", func(t *testing.T) {
+		op_e2e.InitParallel(t)
+		cfg := e2esys.DefaultSystemConfig(t)
+		cfg.DeployConfig.L1GenesisBlockBaseFeePerGas = (*hexutil.Big)(big.NewInt(7))
+
+		cfg.DeployConfig.L2GenesisRegolithTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisCanyonTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisDeltaTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisEcotoneTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisFjordTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisHoloceneTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.L2GenesisIsthmusTimeOffset = new(hexutil.Uint64)
+		cfg.DeployConfig.GasPriceOracleOperatorFeeScalar = 5000
+		cfg.DeployConfig.GasPriceOracleOperatorFeeConstant = 200
+		testFees(t, cfg)
+	})
 }
 
 func testFees(t *testing.T, cfg e2esys.SystemConfig) {
```

Comment:
This test itself would also greatly benefit from a parameterization to also test non-zero state refunds and user deposit transactions.

I'm also wondering, doesn't the proof test seem redundant to this one? We could probably make this test into a proofs test so it also tests the fault proof programs and then scrap the other?

Reply:
Closing as tracked here: https://github.com/ethereum-optimism/optimism/issues/14626